cfme/tests/services/test_service_catalogs.py

```diff
@@ -16,7 +16,8 @@
 pytestmark = [
     pytest.mark.meta(server_roles="+automate"),
     pytest.mark.usefixtures('logged_in', 'vm_name', 'catalog_item', 'uses_infra_providers'),
-    pytest.mark.long_running
+    pytest.mark.long_running,
+    pytest.mark.tier(2)
```

Comment:
Tier 2 : Integration level functional tests . service catalogs for all providers will come under this . No ?? Just trying to understand . If not then what will come under Tier 2 ?